hooks/event-conversion-utils.ts

```diff
@@ -67,13 +68,33 @@ function getType (record: any): CommunityEventType {
   return 'Conference'
 }
 
-function getImage (_record: any): string {
-  const options = [
-    '/images/events/promo-finland-unconference.jpg',
-    '/images/events/promo-vermont.jpg',
-    '/images/events/promo-asia.jpg'
-  ]
-  return options[Math.floor(Math.random() * options.length)]
+function getImage (record: any): string {
+  const fallbackImage = '/images/events/no-picture.jpg'
+  const attachments = record.get(RECORD_FIELDS.image)
+  const imageAttachment = attachments && findImageAttachment(attachments)
+  const imageUrl = imageAttachment && getBestSizeUrl(imageAttachment)
+  return imageUrl || fallbackImage
+}
+
+function findImageAttachment (attachments: any[]): any|null {
+  for (const oneAttachment of attachments) {
+    const isImage = oneAttachment.type.startsWith('image')
+    if (isImage) {
+      return oneAttachment
+    }
+  }
+  return null
+}
+
+function getBestSizeUrl (imageAttachment: any): string {
```

Comment:
For me, this function name is unclear. You are not getting the "best size url" you are getting the url of the image you can find. Correct me if I'm wrong